sdk/iot/core/src/az_iot_sas_token.c

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
I am not sure if it matters, but in our code, the copyright comment looks this way:
```
// Copyright (c) Microsoft Corporation. All rights reserved.
// SPDX-License-Identifier: MIT
```

Reply:
(some other files you're adding also have different license paragraph)

Reply:
Fixed.